src/App.tsx

```diff
@@ -1,10 +1,41 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoItem } from './components/TodoItem';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+import { ErrorNotification } from './components/ErrorNotification';
+import { filterTodos } from './utils/filterTodos';
+import { ErrorType } from './types/ErrorType';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorTodos, setErrorTodos] = useState<ErrorType>(ErrorType.Empty);
+  const [isloadingTodos, setIsLoadingTodos] = useState(false);
```

Comment:
for what this state? you don`t need to process loading in this specification
and don`t need use 'finally' on 29 line